packages/types/src/types.ts

```diff
@@ -399,9 +466,13 @@ export type AssistantPackageJson = PackageJson &
   }>
 
 /**
- * Platforms that can run Assistants.
+ * Assistants can run within Node, or the JavaScriptCore runtime provided by Sketch. This type
+ * enumerates the two possibilities.
  */
-export type Platform = 'sketch' | 'node'
+export enum AssistantRuntime {
+  SketchJavaScriptCore = 'SketchJavaScriptCore',
+  Node = 'Node',
```

Comment:
I think I `Sketch` and `Node` were actually better because:

- `Sketch` = `JavaScriptCore` + some Sketch APIs
- `Node` = `V8` + some Node APIs

… V8 and JavaScriptCore both being JavaScript engines?

Reply:
Ah yes, my b.